app/models/tube_rack/purpose.rb

```diff
@@ -3,11 +3,19 @@
 # The purpose of a tube rack is to hold tubes.
 # Created to hold the size of the tube rack for use when generating manifests.
 class TubeRack::Purpose < Purpose
-  self.default_prefix = 'TR'
```

Comment:
Was this used anywhere?

Reply:
It's used for creating new barcodes, which we don't need to do for tube racks. They have etched barcodes and we use those instead.